DB/DB_SCHEMA.sql

```diff
@@ -0,0 +1,28 @@
+create type category as enum('PHONE','LAPTOP','DESKTOP','AUDIO','TABLET','EARPHONE');
+
+CREATE TABLE PRDT_INFO(
```

Comment:
IMHO, the `INFO` is not clear.

How about..
```sql
CREATE TABLE PRODUCT(
```

Even better with following regular SQL naming convension.

```sql
CREATE TABLE products(
```

Reply:
In my work we always using uppercase for naming table or column.
FXXKing TaiShin trash bank.
I will change my habit , thanks Mr.egg

Reply:
I genuinely hope to avoid using too many abbreviations unless necessary, as they can sometimes be confusing...

BTW, I'm fine with using `PRODUCT` or `PRODUCT_INFO`.